TASK-5/TASK-5/Menu.cs

```diff
@@ -0,0 +1,172 @@
+        {
+            counter++;
+            Console.WriteLine($"{counter}. {user.FullName} || Age is {user.Age} || ID is {user.PassportId}");
+        }
+    }
+
+    private static void DisplayClientPurchases(List<User> users, int userNumber)
+    {
+        decimal totalPrice = 0;
+        foreach (var items in users[userNumber-1].GroceryBag)
+        {
+            Console.WriteLine($"{items.Barcode} || {items.Category} || {items.Price}$ || {items.ItemName}");
+            totalPrice += items.Price;
+        }
+        Console.WriteLine($"Total: {totalPrice}$");
+    }
+    
+    private static void DisplayClientPurchases(List<User> users)
+    {
+        decimal totalPrice = 0;
+        string input;
+        int itemNumber;
+        do
+        {
+            Console.WriteLine("Please, enter User number (starts from 1): ");
+            input = Console.ReadLine();
+        } while (!int.TryParse(input, out itemNumber));
+        
+        foreach (var items in users[itemNumber-1].GroceryBag)
+        {
+            Console.WriteLine($"{items.Barcode} || {items.Category} || {items.Price}$ || {items.ItemName}");
+            totalPrice += items.Price;
+        }
+        Console.WriteLine($"Total: {totalPrice}$");
+    }
+    
+   private static void GenerateUserFromConsole(List<User> users) 
+    {
+        Console.WriteLine("Enter new user's ID");
+        var userPassportIdInput = Console.ReadLine();
+        Guid passportId;
+        while (string.IsNullOrWhiteSpace(userPassportIdInput) || userPassportIdInput.Length != 32 
+                                                              || !Guid.TryParse(userPassportIdInput, out passportId))
+        {
+            Console.WriteLine("User's passport ID cannot be empty and it's length has to be 32 symbols");
+            userPassportIdInput = Console.ReadLine();
+        }
+        
+        if (Validation.SameIdChecker(passportId, users))
+        {
+            users.Add(UserGenerator.CreateUserUsingConsole(passportId));
+            Console.WriteLine("You've added new user successfully");
+        }
+        else
+        {
+            Console.WriteLine("User cannot be added, this ID already exists");
+        }
+    }
+  
+    private static void AddItemToBag(List<User> users)
+    {
+        Console.WriteLine("Choose userNumber to add new item to");
+        var userNumber = Convert.ToInt32(Console.ReadLine());
+        DisplayClientPurchases(users, userNumber);
+        Console.WriteLine("You are in a process of generating item:\n" +
+                          " Please, enter 1 to add a non-alcohol item, 2 for an alcohol item only");
+        var temporaryItem = new Item();
+        switch (Convert.ToInt32(Console.ReadLine()))
+        {
+            case 1:
+                temporaryItem = ItemGenerator.GenerateItem();
+                break;
+            case 2:
+                temporaryItem = ItemGenerator.GenerateAlcoholItem();
+                break;
+        }
```

Comment:
Create a case for manual creation of item(input Barcode, ItemName, Category, Price)